demo/run.bash

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env bash
+
+# Install Nix package manager
+if [[ ! -d "/nix" ]] || [[ ! -d "$HOME/.nix-profile" ]];
+then
+  if [[ -x "$(command -v curl)" ]];
+  then
+    curl https://nixos.org/nix/install | sh
+  else
+    echo "Need 'curl' to install Nix"
+    exit 1
+  fi
+fi
+echo "OK: Nix installed"
+
+# Activate Nix environment
+if [[ ! -x "$(command -v nix-env)" ]] && [[ ! -x "$(command -v nix-shell)" ]];
+then
+  . ~/.nix-profile/etc/profile.d/nix.sh
+fi
+echo "OK: Nix environment activated"
+
+# Install Batsim, Batshed and Batexpe
+if ! nix-env -q 'batsim' &> /dev/null;
```

Comment:
Not necessary to test if already installed: Nix will handle this by doing nothing it is already  intalled and by updating to the last version if necessary.